projects/angular/src/wizard/API.md

```diff
@@ -0,0 +1,42 @@
+# Wizard-clarity - Style API
+
+| CSS Variable Name          | Description                                                        |
+| -------------------------- | ------------------------------------------------------------------ |
+|  --clr-wizard-main-bgcolor | Changes the background color in wizard element.
+|  --clr-wizard-stepnav-bgcolor | Changes the background color of the side navigation bar for all wizard elements..
+|  --clr-wizard-main-text-color | Changes the color of text in form validation for successfully filling information; in alt net, changes the color of text for checkbox text and in step navigation changes the text color of required information to proceed.
+|   --clr-wizard-stepnav-text | Changes the color of text inside the side navigation bar of wizard element for visited items.
+|  --clr-wizard-stepnav-text--active | Changes the text's color for the currently active element inside the side nav bar of wizard element.
+|  --clr-wizard-title-text | Changes the color of the text in the title of a wizard element. This change affects all wizard elements.
+|  --clr-wizard-stepnav-border-size | Changes the side nav bar's border size and is applied to each side nav bar of the wizard element.
+|  --clr-wizard-stepnav-item-border-color | Changes the border color of the side nav bar and is applied to all wizard element side nav bars.
+|  --clr-wizard-stepnav-item-border-color--active | Changes the border color of the side nav bar when element is active and is applied to all wizard element side nav bars.
+|  --clr-wizard-stepnav-active-bgcolor | Changes the background color of an active item in a side navigation bar and applied it to all side navigation bars of wizard element.
+|  --clr-wizard-header-action-color| It changes the color of the text.
+| --clr-wizard-header-action-color--hovered | when the mouse is over it, the text's color changes.
+| --clr-wizard-border-radius | Changes the wizard element's corner radius and applies to all wizard elements..
```

Comment:
typo: .. -> .
please change for the other occurrences below

Reply:
Updated, As suggeted